### PR TITLE
GIT_02.local

### PR DESCRIPTION
**Pull request by RPMFAN**

### CODE REVIEW

**Comment by RPMFAN** [comment]
Changed the source branch and created a new pull request
Let me know if something wrong 
Thanks

File: ayehiazaran/sa2_02.git.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+echo This is my first commit to master branch
```
[the remark this answers is not in the document]
Comment (Webonnix) [reply ayehiazaran/sa2_02.git.sh]: It's just a file to which I added information to before commits

[the remark this answers is not in the document]
Comment (RPMFAN) [reply ayehiazaran/sa2_02.git.sh]: It's just a file to which I added information to before commits

File: ayehiazaran/02_GIT.me [hunk 2]
```diff
@@ -0,0 +1,16 @@
+  97  git clone https://github.com/pluhin/sa.it-academy.by.git && ll
```
[the remark this answers is not in the document]
Comment (RPMFAN) [reply ayehiazaran/02_GIT.me]: Done

**Pull request merged**